main.css

```diff
@@ -168,6 +174,35 @@ html, body
     text-align: left;
 }
 
+#dateSelection
+{
+    display: flex;
+}
+
+button
+{
+    background-color: #fff;
+    font-size: 14px;
+    border: 1px solid lightgrey;
+    margin-left: -1px;
+    padding: 5px 5px;
+}
+
+#dateSelection:first-child
+{
+    margin-left: 1px;
+}
+
+button:hover
+{
+    background-color: #eee;
+}
+
+button:active
+{
+    background-color: #bbb;
+}
+
```

Comment:
The legend items labels don't match up exactly with the checkbox. Might try something like:
```
#typeSelection input[type=checkbox] {
    cursor: pointer;
    margin: 5px;
    padding: 5px;
    vertical-align: -3px;
}
```